scripts/qa_crowbarsetup.sh

```diff
@@ -2891,8 +2895,8 @@ function custom_configuration()
         ;;
     esac
 
-    crowbar $proposal proposal --file=$pfile edit $proposaltype ||\
-        complain 88 "'crowbar $proposal proposal --file=$pfile edit $proposaltype' failed with exit code: $?"
+    crowbar $proposal proposal edit $proposaltype --file=$pfile ||\
```

Comment:
any reason why you changed these 2 lines? nothing about it in the commit message